pygmt/figure.py

```diff
@@ -284,6 +284,27 @@ def show(self, dpi=300, width=500, method="static"):
             img = Image(data=png, width=width)
         return img
 
+    def shift_origin(self, xshift=None, yshift=None):  # pylint: disable=no-self-use
```

Comment:
```suggestion
    def shift_origin(self, xshift=None, yshift=None): 
```

@seisman you still need to call `self._preprocess()` to make sure we're plotting to the right figure. This is not a great design, granted. But the only other I can think of for calling the `figure` module automatically is through a metaclass, which I'm not eager to do.